app/styles/docs.css

```diff
@@ -179,7 +179,7 @@
   /*****************************************************************************/
   /* links */
   & a {
-    @apply text-blue-600 decoration-1 underline-offset-2 hover:underline dark:text-blue-500;
+    @apply text-blue-700 underline decoration-1 underline-offset-2 hover:underline dark:text-blue-500;
```

Comment:
Looks like you added `underline` to all links instead of just `hover`

Reply:
I'm not personally a fan of the underlines from an aesthetic standpoint in this context. There's plenty of reputable sites (e.g., Apple, Vercel, Joshua Comeau's blog, etc) that only use color to indicate a link. I'll defer to @brookslybrand here